api/v1/openapi.yaml

```diff
@@ -3124,6 +3124,7 @@ definitions:
             - HostPort
             - LoadBalancer
             - LocalRedirect
+            - HeadLess
```

Comment:
Sorry, it should be `Headless`, not `HeadLess`.

Source https://kubernetes.io/docs/concepts/services-networking/service/#headless-services